en/full-node.md

```diff
@@ -190,7 +190,7 @@ have an easy-to-use node.
 * Desktop or laptop hardware running recent versions of Windows, Mac OS
   X, or Linux.
 
-* {{site.text.bitcoin_datadir_gb}} gigabytes of free disk space,
+* {{site.text.bitcoin_datadir_gb_pruned}} gigabytes of free disk space,
   accessible at a minimum read/write speed of 100 MB/s.
 
 * 2 gigabytes of memory (RAM)
```

Comment:
Most consumer laptops only have 8GB of RAM. Using 2GB of RAM for bitcoin, what is the minimum memory, if you keep UTXO on disk, and still relay blocks? I read elsewhere the minimum of 1GB, but I want confirmation from someone who runs a full node with minimal memory. (@luke-jr)